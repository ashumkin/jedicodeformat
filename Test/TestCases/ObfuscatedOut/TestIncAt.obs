unit testincat;interface implementation procedure foo;var at:integer;begin at:=2
;inc(at,2);end;end. 