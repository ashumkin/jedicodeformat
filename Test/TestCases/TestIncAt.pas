unit TestIncAt;

interface

implementation

procedure Foo;
var
  at: integer;
begin
 at := 2;
 inc(at, 2);
end;

end.
