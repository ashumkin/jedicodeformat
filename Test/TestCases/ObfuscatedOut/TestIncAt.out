unit testincat;

interface

implementation

procedure foo;
var
  at: integer;
begin
  at := 2;
  Inc(at, 2);
end;

end.
 